cdap-security/src/main/java/io/cdap/cdap/security/auth/ProxyUserIdentityExtractor.java

```diff
@@ -68,19 +70,29 @@ public UserIdentityExtractionResponse extract(HttpRequest request)
     UserIdentity identity = new UserIdentity(userIdentity, UserIdentity.IdentifierType.EXTERNAL,
         new LinkedHashSet<>(), now, now + EXPIRATION_SECS);
 
-    // Parse the access token from authorization header. The header will be in "Bearer" form.
+    // Parse the access token from authorization header. The header will be in "Bearer" form, if external
+    // or "CDAP-Internal" if internal
```

Comment:
nit: replace `Bearer` with `{@link io.cdap.cdap.proto.security.Credential.CredentialType#EXTERNAL_BEARER}` and `CDAP-Internal` with `{@link io.cdap.cdap.proto.security.Credential.CredentialType#INTERNAL}`.

Reply:
Sure, done.